index.js

```diff
@@ -10,8 +10,13 @@
 var isUTF8 = require('is-utf8');
 var isBuffer = require('is-buffer');
 
+function maybeUTF8(buf) {
+  // Only "maybe" because we aren't sniffing the whole buffer
+  return isUTF8(buf.slice(0, 7));
```

Comment:
It can but I don't think it's 100% (hence the "maybe" naming).  This is the same logic as we use in the streaming interface, so it would stay consistent.  But hey, the tests here and in vinyl-fs pass 😛